zipkin-collector-service/src/main/scala/com/twitter/zipkin/collector/Main.scala

```diff
@@ -14,31 +14,35 @@
  *  limitations under the License.
  *
  */
-package com.twitter.zipkin.collector
+import com.twitter.zipkin.thriftscala.{Span => ThriftSpan}
+import com.twitter.finagle.stats.StatsReceiver
+import com.twitter.server.TwitterServer
+import com.twitter.util.{Await, Future}
+import com.twitter.zipkin.cassandra.CassieSpanStoreFactory
+import com.twitter.zipkin.collector.{SpanReceiver, ZipkinQueuedCollectorFactory}
+import com.twitter.zipkin.common._
+import com.twitter.zipkin.receiver.kafka.KafkaSpanReceiverFactory
+import com.twitter.zipkin.storage.WriteSpanStore
+import com.twitter.zipkin.zookeeper.ZooKeeperClientFactory
+import kafka.serializer.Decoder
 
-import com.twitter.logging.Logger
-import com.twitter.ostrich.admin.{ServiceTracker, RuntimeEnvironment}
-import com.twitter.util.Eval
-import com.twitter.zipkin.collector.builder.CollectorServiceBuilder
-import com.twitter.zipkin.BuildProperties
+import com.twitter.zipkin.receiver.kafka.SpanDecoder
 
+object ZipkinKafkaCollectorServer extends TwitterServer
+  with ZipkinQueuedCollectorFactory
+  with CassieSpanStoreFactory
+  with ZooKeeperClientFactory
+  with KafkaSpanReceiverFactory
+{
+  def newReceiver(receive: Seq[ThriftSpan] => Future[Unit], stats: StatsReceiver): SpanReceiver =
+    newKafkaSpanReceiver(receive, stats.scope("kafkaSpanReceiver"), new SpanDecoder())
 
-object Main {
-  val log = Logger.get(getClass.getName)
+  def newSpanStore(stats: StatsReceiver): WriteSpanStore =
```

Comment:
ZipkinCollectorFactory.scala requires newSpanStore and NewReceiver be implemented